tests/core/tools/color.js

```diff
@@ -0,0 +1,264 @@
+/* bender-tags: editor */
+
+
+/**
+ * Basic definitions
+ * HEX 	 -> valid HEX Color Value with # value: #FFF or #FFFFFF
+ * 3-HEX -> HEX but with only 3 characters value: #FFF
+ * 6-HEX -> HEX but with exactly 6 characters value: #FFFFFF
+ * 8-HEX -> HEX but with exactly 8 characters value: #FFFFFF00. Last two characters are for alpha
+ */
```

Comment:
I try to move in a slightly different direction.
Most important are input and output digits in hexadecimal color code.
So I remove `HEX` from test cases names. In each test case, we know exactly how many digits are in input and output. 

There is no need to define so generic valid `HEX` term.
It was - before API was defined to return only `6-HEX` or `8-HEX`.

Hope it will be good.